app/index_controller.go

```diff
@@ -17,3 +19,27 @@ func (app *App) RenderIndex(w http.ResponseWriter, r *http.Request) {
 	}
 	io.WriteString(w, res)
 }
+
+func (app *App) RenderPDFIndex(w http.ResponseWriter, r *http.Request) {
+	tmplList := []string{"./web/views/pdf.html"}
+	// "./web/views/User/Login.html"}
+
+	body := "my body is ready"
+
+	data := &struct {
+		Body string
+	}{body}
+
+	res, err := app.TplParser.ParseTemplate(tmplList, data)
+	if err != nil {
+		app.Log.Info(err)
```

Comment:
what happens if there is an error?Do you continue with generating a blank PDF?Should you not return?